3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -84,6 +84,70 @@ Per i requisiti di sistema e hardware si rimanda al documento #st_v.
 
 
 = Istruzioni d'uso
+== Avvio e Configurazione dell'ambiente
+
+Il software all'avvio si presenta come segue:
+#figure(
+  image("./imgs/avvio.png", width: 60%),
+  caption: [
+    Avvio del software
```

Comment:
Forse cambierei la caption in "Schermata iniziale", ma vedi pure te cosa preferisci